airflow-core/src/airflow/ui/src/i18n/locales/de/admin.json

```
@@ -1,6 +1,161 @@
 {
-  "connections":{
+  "columns": {
+    "description": "Beschreibung",
+    "key": "Name",
+    "name": "Name",
+    "value": "Wert"
+  },
+  "config": {
+    "columns": {
+      "section": "Abschnitt"
+    },
+    "title": "Airflow Konfiguration"
+  },
+  "connections": {
+    "add": "Verbindung hinzufügen",
+    "columns": {
+      "connectionId": "Verbindungs-ID",
+      "connectionType": "Verbindungstyp",
+      "host": "Hostname",
+      "port": "Port"
+    },
+    "connection_one": "Verbindung",
+    "connection_other": "Verbindungen",
+    "delete": {
+      "deleteConnection_one": "1 Verbindung löschen",
+      "deleteConnection_other": "{{count}} Verbindungen löschen",
+      "firstConfirmMessage_one": "Die folgende Verbindung wird gelöscht:",
+      "firstConfirmMessage_other": "Die folgenden Verbindungen werden gelöscht:",
+      "title": "Verbindung löschen"
+    },
+    "edit": "Verbindung bearbeiten",
+    "form": {
+      "connectionIdRequired": "Verbindungs-ID ist erforderlich",
+      "connectionIdRequirement": "Die Verbindungs-ID kann nicht nur aus Leerzeichen bestehen",
+      "connectionTypeRequired": "Die Auswahl eines Verbindungstyps ist erforderlich",
+      "extraFields": "Weitere Felder",
+      "extraFieldsJson": "JSON Ansicht der weiteren Felder",
+      "helperText": "Fehlt ein Verbindungstyp? Stellen Sie sicher, dass die notwendigen Provider-Pakete installiert sind.",
+      "selectConnectionType": "Verbindungstyp auswählen",
+      "standardFields": "Standardfelder"
+    },
+    "noRowMessage": "Keine Verbindungen gefunden",
+    "searchPlaceholder": "Verbindungen suchen",
     "test": "Verbindung testen",
-    "testDisabled": "Das Testen von Verbindungen ist deaktiviert. Der Administrator kann via Konfiguration das Testen freischalten."
+    "testDisabled": "Das Testen von Verbindungen ist deaktiviert. Der Administrator kann via Konfiguration das Testen freischalten.",
+    "typeMeta": {
+      "error": "Fehler beim Abrufen der Liste der Verbindungstypen",
+      "standardFields": {
+        "description": "Beschreibung",
+        "host": "Hostname",
+        "login": "Benutzername",
+        "password": "Passwort",
+        "port": "Port",
+        "url_schema": "Verbindungsschema"
+      }
+    }
+  },
+  "deleteActions": {
+    "button": "Löschen",
+    "modal": {
+      "confirmButton": "Ja, Löschen",
+      "secondConfirmMessage": "Diese Aktion kann nicht rückgängig gemacht werden. ",
```

```suggestion
      "secondConfirmMessage": "Diese Aktion kann nicht rückgängig gemacht werden.",
```

unless the added space was on purpose 🤔

---

Yes, was added on purpose because the "thirdConfimMessage" is appended as bold text and w/o the whitespace the text is glued together.